### DISCUSSION TITLE
xDrip cloud update to Nightscout

### DISCUSSION THREAD

**Discussion by Scizzer** [NightscoutFoundation/xDrip#2405] [Q&A]
I am attempting to upload via cloud through xDrip to Nightscout using xDrip cloud. I get the attached error "unable to do a rest API download" not sure to overcome this problem, any suggestion would be welcome

**Comment by Navid200** [comment]
You have revealed your id and password in that image.  You can delete it.
Can you open a terminal on the virtual machine and show a screenshot of the status page?

**Reply by Scizzer** [reply]
Would do so but not sure how

**Reply by Navid200** [reply]
Did you follow this guide to install?
https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud

**Reply by Scizzer** [reply]
Yes, I followed it, think it is maybe xDrip, not sure. After Google Cloud, I tried NS10BE in desperation, still same outcome, xDrip does not update Nightscout

**Reply by jonprogers** [reply]
I'd confirm your BaseURL in xDrip>Settings>Cloud Upload>Nightscout Sync>Base URL as unless you have made other config updates it should normally be an https url - I think the screenshot you posted earlier had an http url.
Also check that you can load your site in a browser with the part of the base url between (but not including) the @ and /api/v1/.

**Reply by Navid200** [reply]
Did you follow this?
https://navid200.github.io/xDrip/docs/Nightscout/NS_setup

**Reply by Scizzer** [reply]
> I'd confirm your BaseURL in xDrip>Settings>Cloud Upload>Nightscout Sync>Base URL as unless you have made other config updates it should normally be an https url - I think the screenshot you posted earlier had an http url. Also check that you can load your site in a browser with the part of the base url between (but not including) the @ and /api/v1/.

Hi, I am able to open it in a browser, however no glucose levels coming through

**Reply by jonprogers** [reply]
OK, going back to my deleted items the base url in the image you posted was http://\<secret\>@sharkscricket.ns.10de/api/v1/ however the url of the site below is sharkscricket.ns.10be.de - these are different (and I still expect you need https: at the beginning).
If you are getting unauthorised errors then check you have the \<secret\> bit specified correctly - confirm the 'ThisIsMyPassword' bit in @Navid200 instructions here https://navid200.github.io/xDrip/docs/Nightscout/NS_setup.

**Comment by Scizzer** [comment]
![Screenshot_20221030-121429_Chrome](https://user-images.githubusercontent.com/116748110/198873441-ae354585-4ec5-46ed-a241-fc89f1f295b4.jpg)

**Comment by Scizzer** [comment]
The other wiered thing is that the android Nightscout app say the URL is unauthorised, yet when you open it up in a browser, it is authorised, I have removed the code for safety purposes

**Reply by Navid200** [reply]
Make sure you have entered the correct syntax as your base URL in xDrip.
I myself have at times forgotten to place the password in front.

**Comment by Scizzer** [comment]
![Uploading Screenshot_20221030-121933_Nightwatch.jpg…]()

**Comment by Scizzer** [comment]
Hi 

I apologise if I am missing your point, however the current URL addresses in Nightscout and xDrip are attached, all appear to be formatted correty, please confirm .

⁣Regards 
Rob​

On 30 Oct 2022, 13:16, at 13:16, Jon ***@***.***> wrote:
>OK, going back to my deleted items the base url in the image you posted
>was ***@***.***/api/v1/ however the url of
>the site below is sharkscricket.ns.10be.de - these are different (and I
>still expect you need https: at the beginning).
>If you are getting unauthorised errors then check you have the
>\<secret\> bit specified correctly - confirm the 'ThisIsMyPassword' bit
>in @Navid200 instructions here
>https://navid200.github.io/xDrip/docs/Nightscout/NS_setup.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/NightscoutFoundation/xDrip/discussions/2405#discussioncomment-4011573
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>

**Reply by jonprogers** [reply]
From image of error message posted yesterday sharkscricket.ns.**10de** and from the browser image today sharkscricket.ns.**10be.de**

**Reply by Navid200** [reply]
Please don't use email to post here.  
Please edit every one of your posts and delete all the auto quotes.

**Comment by Scizzer** [comment]
It's fixed, invalid URL in the xDrip cloud uploaded

Thanks for all the amazing help 

On 30 Oct 2022, 14:06, at 14:06, Jon ***@***.***> wrote:
>From image of error message posted yesterday
>sharkscricket.ns.**10de**/api and from the browser image today
>sharkscricket.ns.**10be.de**
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/NightscoutFoundation/xDrip/discussions/2405#discussioncomment-4011754
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>
